File: Practice Arrays/codigo_cs/8Difficulty Rating Order.cs
using System;

public class Test8
{
	public static void CheckOrderDifficulty()
	{
		// your code goes here
        int t=int.Parse(Console.ReadLine());
        for (int _ = 0; _ < t; _++){
            int n=int.Parse(Console.ReadLine());
            int[]d=Console.ReadLine().Split().Select(int.Parse).ToArray();
            bool flag=false;
            for (int i = 1; i < n; i++){
                if (d[i-1]>d[i]){
                    Console.WriteLine("No");
                    flag=true;
                    break;
                }
            }
            
            if (!flag) Console.WriteLine("Yes");}
	}
}

File: Practice Arrays/8Difficulty Rating Order.py
t = int(input())

while t > 0:
    n = int(input())
    d = list(map(int, input().split()))
    # Your code goes here
    t -= 1
    for i in range(1,n):
        if d[i-1]>d[i]:
            print("No")
            break
    else:
        print("Yes")


